designer/client/src/list/ListEdit.test.tsx

```diff
@@ -40,10 +40,12 @@ describe('ListEdit', () => {
 
     const $listCaption = screen.getByText('List items')
     const $listHint = screen.getByText('Enter a unique name for your list')
+    const $listType = screen.getByText('List type')
```

Comment:
Not a blocker, but can we add role tests for the radios at some point?

E.g. Here's what we do in **ListContentEdit.test.tsx**

https://github.com/DEFRA/forms-designer/blob/2331da9c3618413dada7df4aaa1c215488cb7f21/designer/client/src/components/FieldEditors/ListContentEdit.test.tsx#L87-L104